src/mixins/utils.js

```diff
@@ -32,12 +32,23 @@ export default {
         };
     },
     methods: {
+        getTableColumns (columns) {
+            return columns.map(col => {
+                return {
+                    ...col,
+                    field: _.snakeCase(col.name),
+                    label: this.$trans(_.snakeCase(col.name)),
+                    align: _.get(col,'align', 'left'),
+                    sortable: _.get(col,'sortable', true)
+                };
+            });
+        },
         setPaginationOptions (response) {
             return {
                 ...pageConfig.default,
                 page: parseInt(_.get(response, 'data.page')),
-                perPage: _.get(response, 'data.perPage'),
-                total: _.get(response, 'data.total'),
+                rowsPerPage: _.get(response, 'data.perPage'),
+                rowsNumber: _.get(response, 'data.total'),
```

Comment:
Why do we want to rename these two fields?

Reply:
@pandafox when we use server-side sorting on click on table header then props send by Quasar Paginate follows above mentioned keys. If we don't use their defaults then we have to override that. 
[https://quasar.dev/vue-components/table#qtable-api](https://quasar.dev/vue-components/table#qtable-api )
![image](https://user-images.githubusercontent.com/775097/114429733-5324dd00-9bdb-11eb-82bf-bcb61329c0e6.png)

Reply:
In which case, we should use that throughout as this is the boiler plate, where we want to define the standard to be used throughout.

@wildtron @cedricabuso thoughts?

Reply:
Yup we should use that for consistnecy 👍